packages/amplify-e2e-core/src/utils/sdk-calls.ts

```diff
@@ -204,6 +204,38 @@ export const describeCloudFormationStack = async (stackName: string, region: str
   );
 };
 
+export const getNestedStackID = async (stackName: string, region: string, logicalId: string) => {
+  const cfnClient = new CloudFormation({ region });
+  return (await cfnClient.describeStackResources({ StackName: stackName, LogicalResourceId: logicalId }).promise()).StackResources[0]
```

Comment:
What is the error if there is no resource with `logicalId` in stack `stackName`? Does the call throw an error? or does it send an empty array?